README.md

```diff
@@ -84,4 +84,18 @@ aws:
   region: eu-west-2
   oidc:
     provider: oidc.eks.my-region.amazonaws.com/id/A1B2C3D4E5F6G7H8
-```
\ No newline at end of file
+```
+
+
+# Install via makefile
+
+```commandline
+make setup
+```
+
+This will create a virtual environment called `venv`, and install the Python
```

Comment:
if we're going to unify making venvs can we name it `.venv`. I'd prefer it was stored up top of file browser away from repository code, e.g. out of the way with .git, .vscode, etc.

Reply:
Venv only needed for Git hooks